generators/app/templates/app/scripts/app.js

```diff
@@ -14,7 +14,7 @@ const sblocksComponents = [
   // simple-blocks implementation
   // https://github.com/lexich/simple-blocks
 ];
-const Modernizr = window.Modernizr;
+// const Modernizr = window.Modernizr;
```

Comment:
I think this show optional adding of modernizr using, like in webpack settings - if you want to use - uncomment.